### PR TITLE
Replay testing CMSSW_12_4_2

### PR DESCRIPTION
**Pull request by germanfgv**
# Replay Request

**Requestor**  
ORM

**Describe the configuration**  
* Release: CMSSW_12_4_2
* Run: 352929
* GTs:
   * expressGlobalTag: 124X_dataRun3_Express_v3
   * promptrecoGlobalTag: 124X_dataRun3_Prompt_v3
   * alcap0GlobalTag: 124X_dataRun3_Prompt_v3
* Additional changes:

**Purpose of the test**  
Test configuration for use in production

**T0 Operations cmsTalk thread**  
We'll create the ticket once the replay starts
[Tier0 Operations cmsTalk Forum](https://cms-talk.web.cern.ch/c/offcomp/compops/tier0/210?)

### CODE REVIEW

**Comment by germanfgv** [comment]
@francescobrivio Please let me know the proper GTs for testing this release

**Comment by tvami** [comment]
@germanfgv I have several comments: 
* please move to el8
* please add the HG alignment as in https://github.com/dmwm/T0/pull/4704/commits/51097b35315713c8ad77c4d574840fbd5079b72f
* please add the PPS ALCARECOs: as in https://github.com/dmwm/T0/pull/4686
* GTs: 124X_dataRun3_Express_v3 and 124X_dataRun3_Prompt_v3 we might change this by the time it gets to the real data taking (we are still waiting for input from the subsystems)

File: etc/ReplayOfflineConfiguration.py [hunk 1]
```diff
@@ -158,7 +158,7 @@
     "CMSSW_11_3_2" : defaultCMSSWVersion['default'],
     "CMSSW_11_3_3" : defaultCMSSWVersion['default'],
     "CMSSW_11_3_4" : defaultCMSSWVersion['default'],
-    "CMSSW_12_0_0" : defaultCMSSWVersion['default'],
+    "CMSSW_12_0_0" : defaultCMSSWVersion['default'], produces
```
Comment (tvami) [inline etc/ReplayOfflineConfiguration.py:161]: this is a typo right?

Comment (germanfgv) [reply etc/ReplayOfflineConfiguration.py:161]: yes. removing

**Comment by germanfgv** [comment]
Here is the monitoring for the replay:
https://monit-grafana.cern.ch/d/t_jr45h7k/cms-tier0-replayid-monitoring?orgId=11&var-Bin=5m&var-ReplayID=220708094924&var-JobType=All&var-WorkflowType=All

**Comment by francescobrivio** [comment]
@germanfgv I am surprised that the repacking of the `ScoutingPF` stream in this replay is not failing, it was failing in the 12_4_0 replay due to change of dataformat from 12_3_X to 12_4_X...what am I missing?

**Comment by germanfgv** [comment]
Now testing 12_4_X in #4716

**Pull request closed without merging**